src/libs/workspace-utils.ts

```diff
@@ -4,6 +4,12 @@ export const cloudProviderTypes: Record<CloudProviderType, CloudProviderType> =
   AZURE: 'AZURE',
   GCP: 'GCP'
 }
+
+export const cloudProviderLabels: Record<CloudProviderType, string> = {
+  AZURE: 'Microsoft Azure',
```

Comment:
These labels are all over the place. I would love to see a cleanup PR that has all the different locations reference these new constants.

https://github.com/DataBiosphere/terra-ui/search?q=%22Microsoft+Azure%22

At the least, could you change the 2 usages in the workspaces directory?

Reply:
I may open a separate PR for this since there's some discussion about when/if to merge the filter.

Reply:
#3663 addresses some of this duplication.